go.mod

```diff
@@ -0,0 +1,62 @@
+module github.com/NYTimes/gizmo
+
+require (
+	cloud.google.com/go v0.26.0
+	contrib.go.opencensus.io/exporter/stackdriver v0.6.0 // indirect
+	github.com/NYTimes/gziphandler v1.0.1
+	github.com/NYTimes/logrotate v0.0.0-20170824154650-2b6e866fd507
+	github.com/NYTimes/sqliface v0.0.0-20180310195202-f8e6c8b78d37
+	github.com/Shopify/sarama v1.17.0
+	github.com/VividCortex/gohistogram v1.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.15.21
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
+	github.com/bradfitz/gomemcache v0.0.0-20180710155616-bc664df96737
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/eapache/go-resiliency v1.1.0 // indirect
+	github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21 // indirect
+	github.com/eapache/queue v1.1.0 // indirect
+	github.com/go-kit/kit v0.7.0
+	github.com/go-logfmt/logfmt v0.3.0 // indirect
+	github.com/go-sql-driver/mysql v1.4.0
+	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/golang/protobuf v1.2.0
+	github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db // indirect
+	github.com/googleapis/gax-go v2.0.0+incompatible // indirect
+	github.com/gorilla/context v1.1.1
```

Comment:
running `go mod why github.com/gorilla/context` shows this being used in the examples, maybe use the regular context.Context instead?

Reply:
Likely a relic of the pre-1.7 days. Def worth killing in another PR.